templates/commands/render_action.go

```diff
@@ -190,65 +193,88 @@ func (n *unknownTemplateKeyError) Is(other error) bool {
 
 var templateKeyErrRegex = regexp.MustCompile(`map has no entry for key "([^"]*)"`)
 
-// "srcRoot" may be a file or directory. "pos" is only used for error messages.
-func copyRecursive(pos *model.ConfigPos, srcRoot, dstRoot string, rfs renderFS, overwrite, dryRun bool) (outErr error) {
-	return fs.WalkDir(rfs, srcRoot, func(path string, de fs.DirEntry, err error) error { //nolint:wrapcheck
+type copyParams struct {
```

Comment:
Even though this struct isn't public, it would be great if we used the godoc-style syntax for documentation for consistency.

Reply:
Will do in a follow-up so I don't lose my precious PR approval bit.